tests/tests/files/bug1588908.js

```diff
@@ -0,0 +1,12 @@
+/**
+ * This tests the regexp literal arrow function from
+ * BrowserTestUtils.waitForDocLoadAndStopIt, which was breaking syntax
+ * highlighting in bug 1588908.
+ */
+
+let isHttp = url => /^https?:/.test(url);
+
+function f(x) {
+  // Not sure why anyone would ever do this, but it's valid JavaScript...
+  return x < /^https?:/;
```

Comment:
Interesting. Even GitHub's syntax highlighter isn't highlighting regexp literals consistently (the colon is orange on this line, but black on line 7).